<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHtmlReader</class>
 <widget class="QWidget" name="CHtmlReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btHome">
       <property name="toolTip">
        <string>Home</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/house.png</normaloff>:/new/icons/icons/house.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+H</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btReload">
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btPrev">
       <property name="toolTip">
        <string>Previous</string>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/leftarrow.png</normaloff>:/new/icons/icons/leftarrow.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btNext">
       <property name="toolTip">
        <string>Next</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/rightarrow.png</normaloff>:/new/icons/icons/rightarrow.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+N</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btShowPanel">
       <property name="toolTip">
        <string>Find</string>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/loupe.png</normaloff>:/new/icons/icons/loupe.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+F</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btZoomIn">
       <property name="toolTip">
        <string>Zoom +</string>
       </property>
       <property name="text">
        <string>Zoom +</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/loupe_plus.png</normaloff>:/new/icons/icons/loupe_plus.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl++</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btZoomOut">
       <property name="toolTip">
        <string>Zoom -</string>
       </property>
       <property name="text">
        <string>Zoom -</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/loupe_minus.png</normaloff>:/new/icons/icons/loupe_minus.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+-</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpCharset">
       <property name="title">
        <string>charset</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="cmbCharset">
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CTextBrowserExt" name="brContent" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>CTextBrowserExt</class>
   <extends>QWidget</extends>
   <header>textbrowserext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
